packages/wrangler/src/api/startDevWorker/ProxyController.ts

```diff
@@ -282,14 +282,14 @@ export class ProxyController extends Controller<ProxyControllerEventMap> {
 
 		try {
 			await this.runtimeMessageMutex.runWith(async () => {
-				assert(this.proxyWorker, "proxyWorker should already be instantiated");
+				const { proxyWorker } = await this.ready.promise;
```

Comment:
Another bug fix?

Reply:
Not a bug fix. Tests pass without this. But leads to better error messages when other tests fail.